packages/tools/examples/videoContourSegmentation/index.ts

```diff
@@ -0,0 +1,484 @@
+    bindings: [
+      {
+        mouseButton: MouseBindings.Primary, // Left Click
+      },
+    ],
+  });
+
+  // Get Cornerstone imageIds and fetch metadata into RAM
+  const imageIds = await createImageIdsAndCacheMetaData({
+    StudyInstanceUID: '2.25.96975534054447904995905761963464388233',
+    SeriesInstanceUID: '2.25.15054212212536476297201250326674987992',
+    wadoRsRoot:
+      getLocalUrl() || 'https://d33do7qe4w26qo.cloudfront.net/dicomweb',
+  });
+
+  // Only one SOP instances is DICOM, so find it
+  const videoId = imageIds.find(
+    (it) => it.indexOf('2.25.179478223177027022014772769075050874231') !== -1
+  );
+
+  // Instantiate a rendering engine
+  const renderingEngineId = 'myRenderingEngine';
+  const renderingEngine = new RenderingEngine(renderingEngineId);
+
+  // Create the viewports
+  const viewportInputArray = [
+    {
+      viewportId: viewportId,
+      type: ViewportType.VIDEO,
+      element: element,
+      defaultOptions: {
+        background: <Types.Point3>[0.2, 0, 0.2],
+      },
+    },
+  ];
+
+  renderingEngine.setViewports(viewportInputArray);
+  toolGroup.addViewport(viewportId, renderingEngineId);
+
+  // Get the stack viewport that was created
+  viewport = <Types.IStackViewport>renderingEngine.getViewport(viewportId);
+
+  // Set the stack on the viewport
+  viewport.setVideo(videoId, 1);
+
+  // Render the image
+  renderingEngine.render();
+  const seconds = (time) => `${Math.round(time * 10) / 10} s`;
+
+  element.addEventListener(Enums.Events.IMAGE_RENDERED, (evt: any) => {
+    const { time, duration } = evt.detail;
+    rangeElement.value = time;
+    rangeElement.max = duration;
+    const timeElement = document.getElementById('time');
+    timeElement.innerText = seconds(time);
+    const remainingElement = document.getElementById('remaining');
+    remainingElement.innerText = seconds(duration - time);
+  });
+
+  // Add a segmentation that will contains the contour annotations
+  segmentation.addSegmentations([
+    {
+      segmentationId,
+      representation: {
+        type: csToolsEnums.SegmentationRepresentations.Contour,
+        data: {
+          // geometryIds may not be used anymore because it will be removed in a
+          // near future but it is still initialized for backward compatibility
+          geometryIds: [],
+        },
+      },
+    },
+  ]);
+
+  // Create a segmentation representation associated to the toolGroupId
+  const segmentationRepresentationUIDs =
+    await segmentation.addSegmentationRepresentations(toolGroupId, [
+      {
+        segmentationId,
+        type: csToolsEnums.SegmentationRepresentations.Contour,
+      },
+    ]);
+
+  // Store the segmentation representation that was just created
+  segmentationRepresentationUID = segmentationRepresentationUIDs[0];
+
+  // Make the segmentation created as the active one
+  segmentation.activeSegmentation.setActiveSegmentationRepresentation(
+    toolGroupId,
+    segmentationRepresentationUID
+  );
```

Comment:
I would be pleasantly surprised if we remove this and it doesn't work, as it should become active internally. If it doesn't, then it might be a bug. We shouldn't have to set a segmentation to be active. Take a look at the labelmap examples, none of them have this logic.

Reply:
It is active internally, but it isn't associated with the toolGroup - I could manually add it, but am just going to leave this code for now.  i copied it direct from another example.